2022-01-14 09:53:30,226 INFO    MainThread:133713 [wandb_setup.py:_flush():71] setting env: {}
2022-01-14 09:53:30,226 INFO    MainThread:133713 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 09:53:30,226 INFO    MainThread:133713 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 09:53:30,227 INFO    MainThread:133713 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_095330-2t18cymg/logs/debug.log
2022-01-14 09:53:30,227 INFO    MainThread:133713 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_095330-2t18cymg/logs/debug-internal.log
2022-01-14 09:53:30,227 INFO    MainThread:133713 [wandb_init.py:init():404] calling init triggers
2022-01-14 09:53:30,228 INFO    MainThread:133713 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-14 09:53:30,228 INFO    MainThread:133713 [wandb_init.py:init():460] starting backend
2022-01-14 09:53:30,228 INFO    MainThread:133713 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-14 09:53:30,244 INFO    MainThread:133713 [backend.py:ensure_launched():216] starting backend process...
2022-01-14 09:53:30,248 INFO    MainThread:133713 [backend.py:ensure_launched():221] started backend process with pid: 133770
2022-01-14 09:53:30,249 INFO    MainThread:133713 [wandb_init.py:init():469] backend started and connected
2022-01-14 09:53:30,251 INFO    MainThread:133713 [wandb_init.py:init():533] updated telemetry
2022-01-14 09:53:30,261 INFO    MainThread:133713 [wandb_init.py:init():563] communicating current version
2022-01-14 09:53:30,614 INFO    MainThread:133713 [wandb_init.py:init():568] got version response 
2022-01-14 09:53:30,614 INFO    MainThread:133713 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-14 09:53:30,777 INFO    MainThread:133713 [wandb_init.py:init():606] starting run threads in backend
2022-01-14 09:53:30,828 INFO    MainThread:133713 [wandb_run.py:_console_start():1810] atexit reg
2022-01-14 09:53:30,829 INFO    MainThread:133713 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-14 09:53:30,829 INFO    MainThread:133713 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-14 09:53:30,830 INFO    MainThread:133713 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-14 09:53:30,830 INFO    MainThread:133713 [wandb_init.py:init():633] run started, returning control to user process
2022-01-14 09:53:32,497 INFO    MainThread:133713 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-14 09:53:32,497 INFO    MainThread:133713 [wandb_run.py:_restore():1752] restore
2022-01-14 09:53:34,893 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 09:53:39,994 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: None
2022-01-14 09:53:40,206 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 09:53:40,790 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 2115
}

2022-01-14 09:53:40,893 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 9453
}

2022-01-14 09:53:40,995 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9453
  total_bytes: 9453
}

2022-01-14 09:53:41,098 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9453
  total_bytes: 9453
}

2022-01-14 09:53:41,202 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9453
  total_bytes: 9453
}

2022-01-14 09:53:41,304 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9453
  total_bytes: 9453
}

2022-01-14 09:53:41,408 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9453
  total_bytes: 9453
}

2022-01-14 09:53:41,510 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9453
  total_bytes: 9453
}

2022-01-14 09:53:41,658 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9453
  total_bytes: 9453
}

2022-01-14 09:53:42,058 INFO    MainThread:133713 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9453
  total_bytes: 9453
}
local_info {
}

2022-01-14 09:53:43,142 INFO    MainThread:133713 [wandb_run.py:_append_files():2180] logging synced files
